app/src/main/java/app/revanced/integrations/youtube/settings/Settings.java

```diff
@@ -57,6 +57,7 @@ public class Settings extends BaseSettings {
     public static final BooleanSetting HIDE_ALBUM_CARDS = new BooleanSetting("revanced_hide_album_cards", FALSE, true);
     public static final BooleanSetting HIDE_ARTIST_CARDS = new BooleanSetting("revanced_hide_artist_cards", FALSE);
     public static final BooleanSetting HIDE_EXPANDABLE_CHIP = new BooleanSetting("revanced_hide_expandable_chip", TRUE);
+    public static final BooleanSetting HIDE_YOODLES = new BooleanSetting("revanced_hide_yoodles", TRUE, true);
```

Comment:
Since the patch may cause unexpected behavior such as the snack bar sliding out, this could be true to prevent users confusing on what happened with their snack bar.